packages/client/src/views/OfficeHome/inProgress/InProgress.tsx

```diff
@@ -142,10 +142,16 @@ export class InProgressComponent extends React.Component<
   }
 
   componentDidMount() {
+    window.addEventListener('popstate', function (event) {
+      window.history.pushState(null, document.title, window.location.href)
+    })
     window.addEventListener('resize', this.recordWindowWidth)
   }
 
   componentWillUnmount() {
+    window.removeEventListener('popstate', function (event) {
+      window.history.pushState(null, document.title, window.location.href)
+    })
```

Comment:
@favour234 For the `removeEventListener` to work, it needs to be supplied with the same function reference that was passed to the `addEventListener`. In this case, as you are using inline functions, their reference won't be the same causing `removeEventListener` to not work properly

Reply:
okay